models/cycle_gan_semantic_model.py

```diff
@@ -239,3 +252,7 @@ def optimize_parameters(self):
         self.optimizer_CLS.zero_grad()
         self.backward_CLS()
         self.optimizer_CLS.step()
+
+    def gaussian(self, in_tensor, stddev):
+        noisy_image = torch.zeros(list(in_tensor.size())).data.normal_(0, stddev).cuda() + in_tensor
```

Comment:
Suggestion
```
noisy_image = torch.normal(0, stddev, size=in_tensor.size()).to(in_tensor.device)
```

Reply:
you mean  `noisy_image = torch.normal(0, stddev, size=in_tensor.size()).to(in_tensor.device) + in_tensor` ?

Reply:
yes, forgot the end of it